src/modules/commands/discord/urbandictionary.ts

```diff
@@ -72,27 +119,22 @@ export default new Command<discord.Message>({
     })
 
     let defN = 0
-    const definitionEmbed = new discord.MessageEmbed()
-      .setColor('#647CC4')
-      .setTitle(`${list[defN].word} (N°${defN + 1}/${list.length})`)
-      .setURL(list[defN].permalink)
-      .setAuthor('Urban dictionary', '', 'https://www.urbandictionary.com')
-      .setThumbnail('https://i.imgur.com/A6nvY85.png')
-      .addFields(
-        {
-          name: 'Definition',
-          value: list[defN].definition.replace(re, '*'),
-        },
-        { name: 'Example', value: list[defN].example.replace(re, '*') },
-      )
-      .addField('Written on', list[defN].writtenOn.substring(0, 11), true)
-      .addField('Thumbs up', list[defN].thumbsUp, true)
-      .addField('Thumbs down', list[defN].thumbsDown, true)
-      .setFooter('Interactive')
-      .setTimestamp()
+
+    let definitionEmbed = genDefinitionEmbed(list[defN], list.length)
 
     const embedMessage = await raw.channel.send({ embed: definitionEmbed })
 
+    if (list.length === 1) {
```

Comment:
This check should be done before you send the embed. Sending it and immediately editing it is unnecessary.